pick/__init__.py

```diff
@@ -21,7 +21,7 @@ class Picker(object):
     :param options_map_func: (optional) a mapping function to pass each option through before displaying
     """
 
-    def __init__(self, options, title=None, indicator='*', default_index=0, multiselect=False, multi_select=False, min_selection_count=0, options_map_func=None):
+    def __init__(self, options, title=None, indicator='*', default_index=0, multiselect=False, multi_select=False, min_selection_count=0, max_selection_count=0, options_map_func=None, pre_select_all=False):
```

Comment:
`max_selection_count` should default to None

Reply:
Ok